src/quoridor/server/receiver.go

```diff
@@ -0,0 +1,40 @@
+package server
+
+import (
+	"encoding/json"
+	"io"
+	"net/http"
+
+	"quoridor/game"
+
+	"github.com/gorilla/mux"
+)
+
+func getGameID(r *http.Request) string {
```

Comment:
I'm not sure that this function is useful... a simple destructuring will also work in one line